checkmate/services/custom_rule.py

```diff
@@ -0,0 +1,60 @@
+from checkmate.checker import url
+from checkmate.models import CustomRule, Reason
+
+
+class CustomRuleService:
+    def __init__(self, db):
+        self._db = db
+
+    def set_block_list(self, text: str) -> list[str]:
+        rules, errors = self._parse_text(text)
+        if not errors:
+            self._set_custom_rules(rules)
+        return errors
+
+    def get_block_list(self) -> str:
+        lines = [self._render_line(rule) for rule in self._db.query(CustomRule).all()]
```

Comment:
We tend to write new code with the newer SQLAlchemy 2.0 style queries


self._db.execute(select(CustomRule)).all() 


But this query is so simple that I don't think it matters.